src/Table.luna

```diff
@@ -126,21 +283,107 @@ class Table:
         cols = self.toList
         columnValueLists = cols.each (_ . slice 0 rowCount . toList)
         JSON.empty . insert "header" self.columnNames . insert "data" columnValueLists
+    
+    # Returns `Table` value with records for which the given predicate returns `True`.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > filteredTable = table.filter (x: x.at "col2" < 14)
+    #
+    # `f`: The predicate (Boolean-valued function).
+    #
+    # `return`: Table with elements satisfying predicate.
+
     def filter f:
         pred = f TableHandle
         self.fromWrapper $ self.ptr.filter pred
+    
+    # Returns `Table` value with columns for which the given predicate returns `True`.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = ["a", "b", "c", "d", "e"]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > filteredTable = table.filterColumns .isNumeric
+    #
+    # `f`: The predicate (Boolean-valued function).
+    #
+    # `return`: Table with columns satisfying predicate.
+
     def filterColumns pred:
         cols = self.toList
         filtered = cols.filter pred
         self.fromColumns filtered
+    
+    # Returns `Table` value with results of applied functions on each of the columns from `self` table.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > filteredTable = table.eachColumn .max
+    #
+    # `pred`: {min, max, mean, median, std, var, sum}.
+    #
+    # `return`: Table with function application results.
+
     def eachColumn pred:
         cols = self.toList
         filtered = cols.each pred
         self.fromColumns filtered
+
+    # Returns `Column` value with results of function `f` application.
```

Comment:
Returns a `Column` created as a result of applying the function `f` to each row of the table.